apps/yarn.lock

```diff
@@ -1738,6 +1744,14 @@ chalk@^0.5.0, chalk@^0.5.1, chalk@~0.5.1:
     strip-ansi "^0.3.0"
     supports-color "^0.2.0"
 
+chalk@^2.0.1:
```

Comment:
I'm just not sure how you ended up with a 2.x version under `@code-dot-org/johnny-five` in your node_modules, when that dependency still resolves to 1.1.3 here in yarn.lock.